script/src/syscalls/load_header.rs

```diff
@@ -93,6 +94,38 @@ impl<'a, DL: DataLoader + 'a> LoadHeader<'a, DL> {
             Source::Group(SourceEntry::HeaderDep) => Err(INDEX_OUT_OF_BOUND),
         }
     }
+
+    fn load_full<Mac: SupportMachine>(
+        &self,
+        machine: &mut Mac,
+        header: &HeaderView,
+    ) -> Result<(u8, u64), VMError> {
+        let data = header.data().as_bytes();
+        store_data(machine, &data)?;
+        Ok((SUCCESS, data.len() as u64))
+    }
+
+    fn load_by_field<Mac: SupportMachine>(
+        &self,
+        machine: &mut Mac,
+        header: &HeaderView,
+    ) -> Result<(u8, u64), VMError> {
+        let field = HeaderField::parse_from_u64(machine.registers()[A5].to_u64())?;
+        let epoch = match self.data_loader.get_block_epoch(&header.hash()) {
+            Some(epoch) => epoch,
+            None => return Ok((ITEM_MISSING, 8)),
```

Comment:
Use `std::mem::size_of<u64>()`, it makes the code easier to read.

Reply:
Ooops this is a mistake, on an item-missing case the second value should be 0, fixing now.